javalang/parser.py

```diff
@@ -1053,7 +1053,10 @@ def parse_interface_method_declarator_rest(self):
         if self.try_accept('throws'):
             throws = self.parse_qualified_identifier_list()
 
-        self.accept(';')
+        if self.would_accept('{'):
+            body = self.parse_block()
+        else:
+            self.accept(';')
 
         return tree.MethodDeclaration(parameters=parameters,
```

Comment:
Could you include `body` in the `MethodDeclaration` initialization here so it is available in the AST? We can default it to `None` above (same as `throws`).